src/constants/text.js

```diff
@@ -6,4 +6,9 @@ export const UI_TEXT = {
   NEXT: 'Next',
   FINISH: 'Finish',
   BACK: 'Back',
+  PRICE: (amount) => `₪${amount}`,
+  NAME: 'Name',
```

Comment:
This is a function and not a constant, functions should be place into a utils folder. 

This for example can be called `convertToShekels` or anything that you would like to call this